pipeline/pipeline_tasks/api_calls/00_01_yelp.py

```diff
@@ -1,39 +1,76 @@
-# -*- coding: utf-8 -*-
 """
-Created on Thu May 04 16:33:48 2017
+This script scrapes Yelp data and obtains the Yelp rating, number of reviews, and url for each business.
+The output of this table is spit into a table: 'sba_sfdo_yelp'
 
-@author: User
+**need to make sure addresses have been cleaned (particularly the city name) to ensure optimal results.**
 """
 
-import pprint
+import argparse
+import os
+import pandas as pd
+from utilities.db_manager import DBManager
+from utilities import util_functions as uf
 import requests
-from bs4 import BeautifulSoup
+from sqlalchemy import create_engine
+import pandas as pd
```

Comment:
This is really nitpicky, but according to PEP8:

> Imports should be grouped in the following order:
> 
> 1. standard library imports
> 2. related third party imports
> 3. local application/library specific imports
> You should put a blank line between each group of imports.

So the import library code should read:

```
import argparse
import os

import pandas as pd
import requests
from sqlalchemy import create_engine

from utilities.db_manager import DBManager
```